### ISSUE TITLE
Explore adding access levels in parallel to default schemas

### ISSUE THREAD

**Issue by jrambla** [ga4gh-beacon/beacon-framework-v2#5]
In the original Beacon v2 inception. we have considered adding the "/access_levels" endpoint [previous spec issue #14](https://github.com/ga4gh-beacon/specification-v2/issues/14).
Although we parked the idea given the complexity introduced by the alternative schemas, we could revisit it leveraging the Model and the fact that it introduces new opportunities, like adding a shadow file to the default schema with the access levels and/or adding that info at entry level into the BeaconMap configuration.
